### PR TITLE
Adds task requests success message to modal

### PR DESCRIPTION
**Pull request by Ajeyakrishna-k**
### Description:
- When a task request is successfully created the modal shows a success message with the link to access the dashboard site task request details page. Where the users can view their task requests
- Minor css fixes

### Dev Tested:
- [x] Yes

### Images/video of the change:
Screenshot 2023-11-06 at 2 58 51 AM

### CODE REVIEW

File: src/components/issues/TaskRequestForm.tsx [hunk 1]
```diff
@@ -58,85 +60,132 @@ const TaskRequestForm: FC<ActionFormProps> = ({
...
+                        <button
+                            className={styles.card__top__button}
+                            type="submit"
+                            disabled={isLoading || !!requestId || !!taskId}
+                            onClick={handleSubmit}
+                        >
+                            Create Request
+                        </button>
+                    </div>
+                </form>
```
Comment (Ajeyakrishna-k) [inline src/components/issues/TaskRequestForm.tsx:155]: These lines were not changed.

File: src/components/issues/TaskManagementModal.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import Modal from '../Modal';
 import TaskRequestForm from './TaskRequestForm';
 import ActionForm from './ActionForm';
 import { TaskData, TaskRequestData } from '@/components/issues/constants';
+import Image from 'next/image';
```
Comment (bhtibrewal) [inline src/components/issues/TaskManagementModal.tsx:8]: why are we using images here, we could find these easily in react-icons

File: src/components/issues/TaskRequestForm.tsx [hunk 3]
```diff
@@ -58,85 +60,132 @@ const TaskRequestForm: FC<ActionFormProps> = ({
...
-                            7,
-                            'formattedDate'
-                        )}
-                        required
-                        onChange={(e) =>
-                            dispatch({ type: 'endsOn', value: e.target.value })
-                        }
+        <div>
+            {!requestId ? (
+                <form className={styles.request_form}>
```
Comment (bhtibrewal) [inline src/components/issues/TaskRequestForm.tsx:70]: please break this component into smaller components, easy to read and understand

**Review by bhtibrewal — CHANGES_REQUESTED** [review]
added comments, please look into them

File: src/components/issues/TaskRequestForm.tsx (hunk 3 above)
Comment (Ajeyakrishna-k) [reply src/components/issues/TaskRequestForm.tsx]: Done.

File: src/components/issues/TaskManagementModal.tsx (hunk 2 above)
Comment (Ajeyakrishna-k) [reply src/components/issues/TaskManagementModal.tsx:8]: These images are from [RDS design system](https://www.figma.com/file/9ZIFD1BFFoO6bHVSSunxGr/Icon-System?type=design&node-id=0-1&mode=design&t=WdiWYzRg3b52rm6R-0). Not sure if we should replace these with similar react icons.

**Review by ajoykumardas12 — APPROVED** [review]
LGTM

**Review by ardourApeX — APPROVED** [review]
LGTM

**Review by shubhamsigdar1 — APPROVED** [review]
great work @Ajeyakrishna-k

File: src/components/issues/TaskManagementModal.tsx (hunk 2 above)
Comment (bhtibrewal) [reply src/components/issues/TaskManagementModal.tsx:8]: yes there are similar icon available

Comment (bhtibrewal) [reply src/components/issues/TaskManagementModal.tsx:8]: it's okay can be updated later

**Review by bhtibrewal — APPROVED** [review]

**Pull request merged**